### PR TITLE
add tailscale bastion

### PR DESCRIPTION
**Pull request by jaxxstorm**
## Adding a new package?

If this pull request adds a new package:

- [ ] The package's schema URL is correct.
- [ ] The package metadata file, if present, contains: 
  - [ ] a supported category (one of `Cloud`, `Infrastructure`, `Network`, `Database`, `Monitoring`, or `Utility`).
  - [ ] a description that explains what the package does.
  - [ ] a valid logo URL that points to a PNG whose dimensions conform to the others in this repo (e.g., 100x100).
  - [ ] a version number prefixed with `v` that corresponds with a valid GitHub release.
- [x] The package repo contains an Overview doc (`/docs/_index.md`) that includes:
  - [x] a brief explanation of what the package is and what it does.
  - [ ] at least one representative example in all supported languages.
  - [x] a front-matter property for the `layout` set to `package`.
- [x] The package repo contains an Installation and Configuration doc (`/docs/installation-configuration.md`) that includes:
  - [x] links to SDKs in all supported languages.
  - [x] a copyable command for installing the resource plugin if necessary.
  - [ ] an example of configuring the provider with `pulumi config set`.
  - [ ] an example of configuring the provider with environment variables.
- [ ] A CODEOWNER has reviewed the PR.
- [ ] Someone from the @pulumi/docs team has reviewed all documentation.

### CODE REVIEW

**Review by AaronFriel — CHANGES_REQUESTED** [review]
If this is AWS specific, I'd like to include "AWS" in the provider name if possible. Whenever we have "X-on-Y" providers or components, including both in the name helps users find the right provider, and ensures that when a "X-on-Z" comes along, we don't have name collisions.

**Comment by jaxxstorm** [comment]
@AaronFriel it is not AWS specific. See https://github.com/lbrlabs/pulumi-tailscale-bastion/tree/main/provider/pkg/provider

File: community-packages/package-list.json [hunk 1]
```diff
@@ -271,6 +271,10 @@
     {
       "repoSlug": "pulumiverse/pulumi-talos",
       "schemaFile": "provider/cmd/pulumi-resource-talos/schema.json"
+    },
+    {
+      "repoSlug": "lbrlabs/pulumi-tailscale-bastion",
+      "schemaFile": "provider/cmd/pulumi-resource-tailscale-bastion/schema.json"
```
Comment (ringods) [inline community-packages/package-list.json:277]: @jaxxstorm The schema file is not in this location. Can you add it?

**Review by ringods — CHANGES_REQUESTED** [review]
@jaxxstorm can you rebase your PR?

File: community-packages/package-list.json (hunk 1 above)
Comment (jaxxstorm) [reply community-packages/package-list.json:277]: Fixed

**Pull request closed without merging**